webpack/webpack5-localization-plugin/src/LocalizationPlugin.ts

```diff
@@ -207,9 +208,9 @@ export class LocalizationPlugin implements WebpackPluginInstance {
                   Constants.JSONP_PLACEHOLDER;
 
                 if (chunkIdsWithStrings.size === 0) {
-                  return this._noStringsLocaleName;
+                  return this._formatLocaleForFilename(this._noStringsLocaleName);
                 } else if (chunkIdsWithoutStrings.size === 0) {
-                  return `" + ${localeExpression} + "`;
+                  return this._formatLocaleForFilename(`" + ${localeExpression} + "`);
```

Comment:
Don't call the function on this value. The input text is raw JavaScript that will be executed by the webpack runtime.

You will instead need to inject it inside of this call:
https://github.com/microsoft/rushstack/blob/main/webpack/webpack5-localization-plugin/src/AssetProcessor.ts#L290